terra-jupyter-base/custom/package-lock.json

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
Where does this come from?

Reply:
its generated when I did `npm install` (so intelliJ would lint for me as I went) to ensure reproducible runs of the command (i.e. we get the same version of eslint). It's generally good to commit it, but I think I'll add it to .gitignore in this case since this isn't a real web app, and we don't want it put on clusters.